htdocs/core/modules/import/import_csv.modules.php

```diff
@@ -960,9 +960,9 @@ public function import_insert($arrayrecord, $array_match_file_to_database, $obji
 								// Note: For extrafield tablename, we have in importfieldshidden_array an entry 'extra.fk_object'=>'lastrowid-tableparent' so $keyfield is 'fk_object'
 								$sqlSelect = "SELECT rowid FROM ".$tablename;
 
-								if (empty($keyfield)) {
+								
 									$keyfield = 'rowid';
-								}
+								
 								$sqlSelect .= " WHERE ".$keyfield." = ".((int) $lastinsertid);
```

Comment:
This looks strange.
 You do 
select rowid where rowid = xxx
to get the value xxx.